test/test_bson.py

```diff
@@ -551,6 +552,71 @@ def test_datetime_encode_decode(self):
         dt2 = decode(encode({"date": dt1}))["date"]
         self.assertEqual(dt1, dt2)
 
+        # Other timestamp formats.
```

Comment:
I think this comment is somehow stuck in the previous indentation block. If you remove the space below this line it should be formatted correctly by pre-commit.